app/assets/javascripts/react_components/admin_dashboard/admin_recommendations.js.jsx

```diff
@@ -180,60 +180,66 @@ var recommendationTypes = {
 var CreateRecommendationPage = React.createClass({
   getInitialState: function () {
     return {
-      books: [], 
-      selectedBooks: [],
-      recommendation_type: recommendationTypes.AUTO,
-      tags: {countries:[2,3,4], languages:[1,2]}
+      recommendationType: RecommendationTypes.CUSTOM,
+      bookTags: [],
+      projectTags: [],
+      selectedBooks: []
     };
   },
   componentDidMount: function () {
-    this._fetchBooks({});
     $('#recommendation-type-toggle').bootstrapSwitch();
     $('#recommendation-type-toggle').on('switchChange.bootstrapSwitch', this._setRecommendationType);
   },
-  _fetchBooks: function (search_data) {
-    $.ajax({
-      url: "/admin/recommendations/display_books",
-      dataType: 'json',
-      data: search_data,
-      success: function (data) {
-        this.setState({books: data});
-      }.bind(this),
-      error: function (xhr, status, err) {
-        console.error(this.props.url, status, err.toString());
-      }.bind(this)
-    });
+  _setRecommendationType: function (event, state) {
+    if (state) {
+      this.setState({recommendationType: 1});
```

Comment:
Probably want to use the constants you defined in line 175 here.

HI WILL

PHC++